apps/docs/src/app/core/component-docs/wizard/examples/wizard-branching-example.component.html

```diff
@@ -1,93 +1,119 @@
-<fd-wizard>
-    <fd-wizard-navigation>
-        <ul fd-wizard-progress-bar>
-            <li fd-wizard-step [status]="step1status" (statusChange)="step1status = $event" label="Step 1: One Line">
-                <fd-wizard-step-indicator glyph="accept"></fd-wizard-step-indicator>
-                <fd-wizard-content>
-                    <div fd-form-item>
-                        <label fd-form-label for="address-1">Address Line 1</label>
-                        <input fd-form-control type="text" id="address-1" />
-                    </div>
-                    <div fd-form-item>
-                        <label fd-form-label for="address-2">Address Line 2</label>
-                        <input fd-form-control type="text" id="address-2" />
-                    </div>
-                    <div fd-form-item>
-                        <label fd-form-label for="address-3">City, State, and Zip</label>
-                        <input fd-form-control type="text" id="address-3" />
-                    </div>
-                    <fd-wizard-next-step>
-                        <button fd-button [compact]="true" fdType="emphasized" (click)="goToStep(2)">
-                            Go to the payment selection page
-                        </button>
-                    </fd-wizard-next-step>
-                </fd-wizard-content>
-            </li>
-            <li
-                fd-wizard-step
-                branching="true"
-                [status]="step2status"
-                (statusChange)="step2status = $event"
-                label="Step 2: Payment Selection"
-            >
-                <fd-wizard-step-indicator>2</fd-wizard-step-indicator>
-                <fd-wizard-content>
-                    Please select a payment type:
-                    <fd-radio-button
-                        value="bank"
-                        id="radio-1"
-                        name="radio-name-1"
-                        [(ngModel)]="paymentSelection"
-                        (ngModelChange)="paymentSelectionChanged(confirmationDialog)"
-                    >
-                        Bank Transfer
-                    </fd-radio-button>
-                    <fd-radio-button value="credit" id="radio-2" name="radio-name-2" [(ngModel)]="paymentSelection">
-                        Credit Card
-                    </fd-radio-button>
-                    <fd-wizard-next-step>
-                        <button
-                            [disabled]="!paymentSelection"
-                            fd-button
-                            [compact]="true"
-                            fdType="emphasized"
-                            (click)="goToStep(3)"
+<button fd-button (click)="enterFullscreenExample()">Click to open full screen</button>
+
+<div cdkScrollable class="docs-wizard-example-overlay" #overlay>
```

Comment:
Why don't you use the Dialog for full screen Wizard?

Reply:
Well it is not meant to be used in a dialog, so I didn't want to give the impression that it is, so I decided to go the same route that the flexible column layout docs are used.  The mobile wizard is used in a dialog and I'll use a dialog in that example